plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -229,8 +234,13 @@ public function migrate( $args = array(), $assoc_args = array() ) {
 
 		return WP_CLI::success(
 			sprintf(
-				/* Translators: %1$d is the number of migrated orders. */
-				_n( '%1$d order was migrated, in %2$f seconds', '%1$d orders were migrated in %2$f seconds', $processed, 'woocommerce' ),
+				/* Translators: %1$d is the number of migrated orders and %2$d is the execution time in seconds. */
+				_n(
+					'%1$d order was migrated in %2$d seconds.',
+					'%1$d orders were migrated in %2$d seconds.',
+					$processed,
+					'woocommerce'
+				),
```

Comment:
- Adjusted the translator comment.
- Changed the format of the execution time in seconds from float to integer.